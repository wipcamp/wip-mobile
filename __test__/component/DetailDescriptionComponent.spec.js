/**
 * @jest-environment node
 */

import React from 'react'
import renderer from 'react-test-renderer'

import DetailDescription from '../../components/DetailDescriptionComponent'

it('rendering', () => {
    const tree = renderer
        .create(
            <DetailDescription>Test</DetailDescription>
        )
        .toJSON()
    expect(tree).toMatchSnapshot()
})