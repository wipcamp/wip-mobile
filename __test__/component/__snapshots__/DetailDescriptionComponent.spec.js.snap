// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<View
  style={
    Array [
      Object {
        "paddingBottom": 10,
        "paddingTop": 10,
      },
      Object {
        "paddingBottom": 15,
      },
      Object {
        "paddingLeft": 10,
        "paddingRight": 10,
      },
      Object {
        "backgroundColor": "#FFF",
      },
      Object {
        "borderBottomWidth": 0.4,
        "borderColor": "#C8C8C8",
      },
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontFamily": "Kanit",
        },
        Object {
          "fontSize": 18,
        },
      ]
    }
  >
    รายละเอียด :
  </Text>
  Test
</View>
`;
